.circleci/config.yml

```diff
@@ -103,6 +103,14 @@ jobs:
           # hardcoded in karma-webpack
           path: /tmp/_karma_webpack_
           destination: artifact-file
+  test_integration:
```

Comment:
what about we call them puppeteer? We already have integration and e2e tests, it will be confusing otherwise.